assets/pages/new-contributions-landing/contributionsLandingInit.js

```diff
@@ -84,6 +85,9 @@ const init = (store: Store<State, Action, Function>) => {
   selectDefaultPaymentMethod(state, dispatch);
   initialisePaymentMethods(state, dispatch);
   initialiseSelectedAnnualAmount(state, dispatch);
+  storage.setSession('contributionType', state.page.form.contributionType);
+  storage.setSession('paymentMethod', state.page.form.paymentMethod);
```

Comment:
what are these for?

Reply:
It's used in various places. We already set these when they change, this just sets them on initialisation too